spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/StreamDefinitionController.java

```diff
@@ -16,10 +16,19 @@
 
 package org.springframework.cloud.dataflow.server.controller;
 
+import java.util.ArrayList;
```

Comment:
Sorry. I will fix this. The `spring-cloud-stream` project now uses [this import settings] (https://github.com/spring-cloud/spring-cloud-stream/blob/master/src/checkstyle/checkstyle.xml#L85) and I got it continued here through my IDE settings. I think we need to decide on the common settings across the projects.